src/qmxgraph/api.py

```diff
@@ -59,10 +70,16 @@ def insert_vertex(self, x, y, width, height, label, style=None, tags=None, id=No
             interaction with cells in a graph.
         :param Optional[str] id: The id of the edge. If omitted (or non
             unique) an id is generated.
+        :param bool adjust_xy_coordinates: A flag indicating the received X and Y
```

Comment:
That's strange, usually while creating objects, coordinates are always given in world coordinates, so screen/view coordinates aren't taken in account.

I think it is worth adding to this comment explaining the reasons and when this flag would be used.

Reply:
I agree that is a bit confusing.
It was probably made this way (receive screen coordinates) to make the mouse drop handling easier.
I could add another method instead adding a flag (`insertVertexWorldCoordinates`?).

Reply:
As we discussed, let's leave it like this for now, and create a new technical debit task to refactor this. 

We discussed two options:

1. User no longer passes a `tuple[float, float]`, but passes instead `WorldPoint` or `ViewPoint`/`ScreenPoint` objects. 
2. Assumes every single coordinate is world coordinates, and provide two functions that can convert from/to world coordinates.

In the end we think 2) is simpler and better in the long run, as 1) requires some custom translation of the custom Python objects.

Reply:
@prusse-martin make sure to create a task for the refactoring, and schedule it for out next sprint.